packages/utils/src/route.js

```diff
@@ -173,6 +174,18 @@ export const createRoutes = function createRoutes ({
         }
       }
     })
+    if (typeof trailingSlashes !== 'undefined') {
+      route.pathToRegexpOptions = { strict: true }
+    }
+
+    if (trailingSlashes) {
+      route.path = route.path.endsWith('/') ? route.path : `${route.path}/`
+    }
+
+    if (trailingSlashes === false) {
```

Comment:
Maybe we can make at an `else if`? This would save branch check-in cases that `trailingSlashes` is true + makes it more readable.

Reply:
How about ?
```js
if(trailingSlashes !== undefined) {
  const trailing = trailingSlashes ? '/' : ''
  route.path = route.path.replace(/\/+$/, '') + trailing
}
```